JSDemos/Demos/DataGrid/ExcelJSExportImages/Angular/app/app.component.ts

```diff
@@ -47,7 +47,9 @@ export class AppComponent {
 
             worksheet.getRow(excelCell.row).height = 90;
             worksheet.addImage(image, {
+              // @ts-expect-error See https://github.com/exceljs/exceljs/issues/1747
               tl: { col: excelCell.col - 1, row: excelCell.row - 1 },
+              // @ts-expect-error See https://github.com/exceljs/exceljs/issues/1747
               br: { col: excelCell.col, row: excelCell.row },
             });
```

Comment:
Let's remove ```ts-expect-error``` and cast these objects to the ```Anchor``` type instead:

```suggestion
           // NOTE: These objects casting to the Anchor type manually because of this issue: 
          //  https://github.com/exceljs/exceljs/issues/1747
           worksheet.addImage(image, {
              tl: { col: excelCell.col - 1, row: excelCell.row - 1 } as Anchor,
              br: { col: excelCell.col, row: excelCell.row } as Anchor,
            });
```

I checked it locally, works fine :)